app/controllers/api/v0/vendors_controller.rb

```diff
@@ -1,4 +1,9 @@
 class Api::V0::VendorsController < ApplicationController
+  def index
+    market = Market.find(params[:market_id])
+    render json: VendorSerializer.new(market.vendors)
+  end
+
   def show
     vendor = Vendor.find(params[:id])
     render json: VendorSerializer.new(vendor)
```

Comment:
A refactor idea we can probably implement is creating a private method to find a vendor, market, etc so we won't have to keep creating them in our action.